SR2021/2020-10-29-sr2021-announcement.md

```diff
@@ -0,0 +1,26 @@
+---
+to: Potential team leaders
+subject: Announcing Student Robotics 2021
+---
+
+Hello there!
+
+I'm pleased to announce some details for Student Robotics 2021.
```

Comment:
```suggestion
We're pleased to announce some details for Student Robotics 2021.
```

Will this email be sent from a personal account or from teams@ (or similar)? If from a shared mailbox and there's no no signature explicitly naming a person feel we should go for 'We' rather than 'I'